### ISSUE TITLE
ScriptUtils.executeDatabaseScript does not support PostgreSQL create function

### ISSUE THREAD

**Issue by lwielek** [testcontainers/testcontainers-java#946]
The utility will fail on a PostgreSQL script, that creates a function.
Why ? Because it splits script by `;` separator into a list of invalid statements.
Example:
```
CREATE FUNCTION jsonb_contains_any(jsonb, CHARACTER VARYING) RETURNS BOOLEAN AS $$
    SELECT $1 ?| string_to_array($2, ',');
$$ LANGUAGE SQL;
```

```
Failed to execute database script from resource [
...
]
	at org.testcontainers.ext.ScriptUtils.executeDatabaseScript(ScriptUtils.java:276)
	at org.testcontainers.ext.ScriptUtils.executeDatabaseScript(ScriptUtils.java:214)
...
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ArrayIndexOutOfBoundsException
	at java.lang.System.arraycopy(Native Method)
	at java.lang.AbstractStringBuilder.append(AbstractStringBuilder.java:570)
	at java.lang.StringBuilder.append(StringBuilder.java:190)
	at org.postgresql.core.Parser.parseSql(Parser.java:1020)
	at org.postgresql.core.Parser.replaceProcessing(Parser.java:974)
	at org.postgresql.core.CachedQueryCreateAction.create(CachedQueryCreateAction.java:43)
	at org.postgresql.core.QueryExecutorBase.createQueryByKey(QueryExecutorBase.java:304)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:283)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:269)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:265)
	at org.testcontainers.jdbc.JdbcDatabaseDelegate.execute(JdbcDatabaseDelegate.java:42)
	at org.testcontainers.delegate.AbstractDatabaseDelegate.execute(AbstractDatabaseDelegate.java:34)
	at org.testcontainers.ext.ScriptUtils.executeDatabaseScript(ScriptUtils.java:263)
	... 34 more
```

**Comment by rnorth** [comment]
I've been meaning to reawaken #579 which would solve this. Sorry I've let it languish. I'll go ahead with that PR soon.

**Comment by rnorth** [comment]
Released in [1.10.4](https://github.com/testcontainers/testcontainers-java/releases/tag/1.10.4)!
